src/test/java/LoginPage.java

```diff
@@ -0,0 +1,24 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends BasePage {
+
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    By emailField = By.cssSelector("input[type='email']");
+    By passwordField = By.cssSelector("input[type='password']");
+    By submitBtn = By.cssSelector("button[type='submit']");
+
+    public void provideEmail(String email) {
+        findElement(emailField).sendKeys(email);
+    }
+    public void providePassword(String password) {
+        findElement(passwordField).sendKeys(password);
+    }
+
+    public void clickSubmit() {
+        findElement(submitBtn).click();
+    }
+}
```

Comment:
A package in Java is used to group related classes. 

For example package "pages" - Includes the Page Objects (or Page Classes)


For example

![image](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/3adf1382-7810-4124-bfbc-8e5b901a293f)